### DISCUSSION TITLE
Support independent SubViews Buttons

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5646] [Feature Requests]
At present, the subview is rendered as read-only if the relationship is independent.

This restriction should be lifted for SubViews that are displayed as a button, as they display the related record in a dialog that has separate save and delete buttons.

See also:
https://github.com/specify/specify7/issues/725
https://github.com/specify/specify7/issues/2006
https://github.com/specify/specify7/pull/2155#pullrequestreview-1109852516

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/appraisal-co-subform-not-allowing-cos-to-be-added/1776/2
